dataset/config.py

```diff
@@ -0,0 +1,182 @@
+""" Config class"""
+
+from copy import deepcopy
+
+class Config:
```

Comment:
Make sure that it supports ** (double star) and looks as similar to `dict` as possible (__setitem__, __len__, __iter__, update, etc).